### PR TITLE
wire the new k8s staging endpoint for the testing frost-web

### PR DESCRIPTION
**Pull request by WesleyCharlesBlake**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes

point the testing netlify frost-web to the K8s staging infrastructure.

### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

File: _redirects.testing [hunk 1]
```diff
@@ -1,2 +1,2 @@
-/api/*  https://api-testing.frost.po.et/:splat  200
+/api/*  https://frost-api.staging.po.et/:splat  200
```
Comment (wzalazar) [inline _redirects.testing]: @WesleyCharlesBlake 
what do you think  if we change the file name to `_redirects.staging` or create a new one?

Comment (WesleyCharlesBlake) [reply _redirects.testing]: Could do. But I'm not sure on how we would get netlify to create an environment from that? If all I need to do is is create a new redirects file then I will do so.

**Comment by WesleyCharlesBlake** [comment]
@wzalazar @lautarodragan : I have created the  staging redirects file here, I've seen package.json already has the script that passes the environment to webpack. But Im not sure on what needs to change on Netflify. Do I just need to create a staging branch for this to take effect?

**Comment by WesleyCharlesBlake** [comment]
closing the PR as I will open up a new PR on staging

**Pull request closed without merging**